api/working-with-extensions/publishing-extension.md

```diff
@@ -87,81 +87,110 @@ First off, follow the documentation to [create your own organization](https://le
 
 1. Click **Create**.
 
-    You'll be presented with your newly created Personal Access Token. **Copy** it to the safe location, you'll need it to [create a publisher](#create-a-publisher).
+    You'll be presented with your newly created Personal Access Token. **Copy** it to the safe location, you'll need it to [create a publisher](#creating-a-publisher).
 
-### Create a publisher
+### Creating a publisher
 
-A **publisher** is an identity who can publish extensions to the Visual Studio Code Marketplace. Every extension needs to include a `publisher` name in its [`package.json` file](/api/references/extension-manifest).
+A **publisher** is an identity that can publish extensions to the Visual Studio Code Marketplace. Every extension should have a `publisher` name in its [`package.json` file](/api/references/extension-manifest).
```

Comment:
I would leave text since it is a requirement not a recommendation

Reply:
If this is a requirement, maybe it would be even better to say 'must include'?